tests/utils/test_cache_utils.py

```diff
@@ -585,12 +597,25 @@ def test_offloaded_cache_uses_less_memory_than_dynamic_cache(self):
         }
         original = GenerationConfig(**common)
         offloaded = GenerationConfig(cache_implementation="offloaded", **common)
-        torch.cuda.reset_peak_memory_stats(device)
+        if device.type == "cuda":
```

Comment:
Instead of repeating the same if else just do it once like this
```python
if device.type == "cuda":
   torch_accelerator_module = torch.cuda
elif  device.type == "xpu":
   torch_accelerator_module = torch.xpu
```

Then you can just call e.g. `torch_accelerator_module.max_memory_allocated(device)` etc. in the rest of the code.

Reply:
yes, it's better, thx.